back-end/lms/serializers/reference_book.py

```diff
@@ -0,0 +1,61 @@
+from rest_framework.serializers import (Serializer, SerializerMethodField)
+
+from lms.models import (
+    StudentPost,
+    TeacherPost,
+    Milgroup,
+    Program,
+    Skill,
+    Rank,
+)
+
+from lms.serializers.serializers import (MilgroupSerializer, ProgramSerializer,
+                                         RankSerializer, SkillSerializer,
+                                         TeacherPostSerializer,
+                                         StudentPostSerializer)
+
+
+# pylint: disable=abstract-method
+class ReferenceBookSerializer(Serializer):
+    milgroups = SerializerMethodField(read_only=True)
+    programs = SerializerMethodField(read_only=True)
+    ranks = SerializerMethodField(read_only=True)
+    skills = SerializerMethodField(read_only=True)
```

Comment:
Just create blank `def create()...` and `def update()...` methods like this:
`
def create(self, validated_data):
        pass

    def update(self, instance, validated_data):
        pass
`